tiny_dnn/core/framework/tensor_utils.h

```diff
@@ -68,55 +68,39 @@ void print_pack(std::ostream &out, T t, U u, Args... args) {
   print_pack(out, u, args...);
```

Comment:
@edgarriba I believe I've fixed it *and* got rid of most of part of variadic templates at the same time